app/models/site.rb

```diff
@@ -9,13 +9,15 @@ class Site < ActiveRecord::Base
 
   delegate :announcement_text, :marketing_text, :featured_researcher,
            :announcement_text=, :marketing_text=, :featured_researcher=,
+           :about_page, :about_page=,
```

Comment:
Please add spec coverage for the changes to this file in `spec/features/site_spec.rb`. You can modify the "updates the homepage" feature spec for this.